Wrong minimal required Maven version for plugins

```
mvn -V help:help

Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T20:33:14+02:00)
Maven home: /Users/slachiewicz/Downloads/apache-maven-3.5.4
Java version: 20, vendor: Azul Systems, Inc., runtime: /Library/Java/JavaVirtualMachines/zulu-20.jdk/Contents/Home
Default locale: en_PL, platform encoding: UTF-8
OS name: "mac os x", version: "13.2.1", arch: "aarch64", family: "mac"
[INFO] Scanning for projects...
[INFO]
[INFO] -------------------< org.codehaus.mojo:mojo-parent >--------------------
[INFO] Building MojoHaus Parent 75-SNAPSHOT
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 0.215 s
[INFO] Finished at: 2023-03-30T01:08:16+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-help-plugin:3.4.0:help (default-cli) on project mojo-parent: The plugin org.apache.maven.plugins:maven-help-plugin:3.4.0 requires Maven version 3.6.3 -> [Help 1]
```

same reported by versions plugin

```
~/Downloads/apache-maven-3.5.4/bin/mvn versions:display-plugin-updates
[INFO] Scanning for projects...
[INFO]
[INFO] -------------------< org.codehaus.mojo:mojo-parent >--------------------
[INFO] Building MojoHaus Parent 75-SNAPSHOT
[INFO] --------------------------------[ pom ]---------------------------------
[INFO]
[INFO] --- versions-maven-plugin:2.15.0:display-plugin-updates (default-cli) @ mojo-parent ---
[INFO]
[INFO] The following plugin updates are available:
[INFO]   maven-scm-publish-plugin ........................... 3.1.0 -> 3.2.1
[INFO]   maven-site-plugin .............................. 3.12.1 -> 4.0.0-M6
[INFO]   org.codehaus.mojo:flatten-maven-plugin ............. 1.3.0 -> 1.4.1
[INFO]   org.codehaus.mojo:license-maven-plugin ................ 1.9 -> 1.20
[INFO]
[INFO] All plugins have a version specified.
[INFO]
[INFO] Project requires minimum Maven version for build of: 3.2.5
[INFO] Plugins require minimum Maven version of: 3.6.3
[INFO]
[ERROR] Project requires an incorrect minimum version of Maven.
[ERROR] Update the pom.xml to contain maven-enforcer-plugin to
[ERROR] force the Maven version which is needed to build this project.
[ERROR] See https://maven.apache.org/enforcer/enforcer-rules/requireMavenVersion.html
[ERROR] Using the minimum version of Maven: 3.2.5
[INFO]
[INFO] Require Maven 3.5.4 to use the following plugin updates:
[INFO]   org.codehaus.mojo:license-maven-plugin ............... 1.9 -> 2.0.0
[INFO]
[INFO] Require Maven 3.6.3 to use the following plugin updates:
[INFO]   maven-help-plugin ........................................... 3.4.0
````

--- a/pom.xml
+++ b/pom.xml
@@ -197,7 +197,7 @@
     <minimalJavaBuildVersion>${mojo.java.target}</minimalJavaBuildVersion>
     <recommendedJavaBuildVersion>${minimalJavaBuildVersion}</recommendedJavaBuildVersion>
 
-    <mavenVersion>3.2.5</mavenVersion>
+    <mavenVersion>3.6.3</mavenVersion>
     <minimalMavenBuildVersion>${mavenVersion}</minimalMavenBuildVersion>
 
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>

